### PR TITLE
vlang 0.1.24 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Many thanks to Vitor Oliveira for their work in #44657 and #44138

### CODE REVIEW

File: Formula/vlang.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  def caveats
+    <<~EOS
+      To use the graphics modules, install glfw and freetype:
+        brew install glfw freetype
+
+      To use the http module, install OpenSSL:
+        brew install openssl
+    EOS
+  end
```
Comment (SMillerDev) [inline Formula/vlang.rb]: This should be in the documentation

File: Formula/vlang.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    end
+    libexec.install "thirdparty", "vlib"
+    if build.head?
+      libexec.install "cmd"
+    else
+      libexec.install "tools"
+    end
+    bin.install_symlink libexec/"v" => "v"
+    pkgshare.install "examples"
+  end
```
Comment (SMillerDev) [inline Formula/vlang.rb]: This is way too complex. Please drop head support.

File: Formula/vlang.rb (hunk 1 above)
Comment (SeekingMeaning) [reply Formula/vlang.rb]: Is it okay if it’s documented in the readme?

Comment (SMillerDev) [reply Formula/vlang.rb]: I don't mind where it's documented but caveats aren't the place for this.

Comment (SeekingMeaning) [reply Formula/vlang.rb]: Ah okay, it's updated now.

**Comment by SeekingMeaning** [comment]
Should this be renamed to `v-lang`?

**Pull request closed without merging**